chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -156,10 +156,10 @@
 
 ruby_block "Ensure /etc/sudoers.d is included in sudoers" do
   block do
-    f = Chef::Util::FileEdit.new("/etc/sudoers")
-    f.insert_line_if_no_match(/^#includedir \/etc\/sudoers.d\/?$/,
-                              "#includedir /etc/sudoers.d")
-    f.write_file
+    sudoers = "/etc/sudoers"
+    if File.readlines(sudoers).none? { |l| l.start_with? "#includedir /etc/sudoers.d" }
+      File.open(sudoers, "a") { |f| f.puts "#includedir /etc/sudoers.d" }
```

Comment:
please add `\n` in front of/before the line to make sure we handle missing EOL @ EOF correctly.

Reply:
done